apps/emqx_mq/src/emqx_mq_schema.erl

```diff
@@ -56,8 +56,44 @@ fields(mq) ->
                 required => true,
                 desc => ?DESC(find_queue_retry_interval),
                 importance => ?IMPORTANCE_MEDIUM
+            })},
+        {auto_create,
+            mk(ref(auto_create), #{
+                required => true,
+                desc => ?DESC(auto_create),
+                default => #{
+                    <<"regular">> => #{<<"enable">> => false},
+                    <<"lastvalue">> => #{<<"enable">> => true}
+                }
             })}
     ];
+fields(auto_create) ->
+    [
+        {regular,
+            mk(ref(auto_create_regular), #{required => true, desc => ?DESC(auto_create_regular)})},
+        {lastvalue,
+            mk(ref(auto_create_lastvalue), #{required => true, desc => ?DESC(auto_create_lastvalue)})}
```

Comment:
maybe `union([false, ref(auto_create_latvalue)])` and set default value to `false`.